pkg/controllers/node/etcd.go

```diff
@@ -0,0 +1,144 @@
+package node
```

Comment:
I just moved the existing logic into this file to keep the `etcd` and `kdd` logic separate.